pkg/virtual/workspaces/auth/cache.go

```diff
@@ -215,6 +218,8 @@ func NewAuthorizationCache(
 
 		reviewer: reviewer,
 		skip:     &neverSkipSynchronizer{},
+
+		watchers: []CacheWatcher{},
```

Comment:
nit: slices nearly never need initialization. Append and range work fine with nil.

Reply:
fixing in follow-up